src/lib.rs

```diff
@@ -449,14 +449,33 @@ fn parse(context: &mut BindgenContext) {
             "How did this happen?");
 }
 
-/// Get the version number (e.g. 3.9.0) of Clang in String form
-pub fn clang_version() -> String {
-    clang::extract_clang_version()
-        .expect("Could not retrieve Clang version!")
-        .split(' ')
-        .nth(2)
-        .expect("Could not parse Clang version!")
-        .to_string()
+/// Extracted Clang version data
+#[derive(Debug)]
+pub struct ClangVersion {
+    /// major semvar
+    pub major: u32,
```

Comment:
Instead of `major` and `minor`, let's use an `Option<(u32, u32)>`¸ and always return the full version (`clang_version() -> ClangVersion`) so we can log it regardless, even if we can't find one of these. I wouldn't care `extract_clang_version` returning just `String`, even if it's empty.